src/sentry/lang/native/applecrashreport.py

```diff
@@ -162,7 +162,10 @@ def get_binary_images_apple_string(self):
             return ""
         binary_images = map(
             lambda i: self._convert_debug_meta_to_binary_image_row(debug_image=i),
```

Comment:
`_convert_debug_meta_to_binary_image_row` also directly grabs `"image_addr"` from the debug image:

```
image_addr = parse_addr(debug_image["image_addr"]) + slide_value
```

should this be updated as well? it looks like that'd still trigger the same key error. is there some sort of implementation detail with `map` that prunes these entries before that error is thrown thanks to this new filter?

Reply:
the new filter added will make sure that entries on which `_convert_debug_meta_to_binary_image_row` would fail are filtered out, avoiding it from crashing.